my_list.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'Enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*items)
+    @list = items
+  end
+end
```

Comment:
- [ ] Kindly make sure to implement the `each` method and as the previous review mentioned consider implementing the `each` functionality on your own and avoid using the existing ruby method `each()`